### PR TITLE
feat(wizard): winforms

### PR DESCRIPTION
**Pull request by bruno-garcia**
// @Tyrrrz

### CODE REVIEW

File: src/wizard/dotnet/winforms.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+{
+    [STAThread]
+    static void Main()
+    {
+        // Init the Sentry SDK
+        SentrySdk.Init("___PUBLIC_DSN___");
+        // Configure WinForms to throw exceptions so Sentry can capture them.
+        Application.SetUnhandledExceptionMode(UnhandledExceptionMode.ThrowException);
+
+        // Any other configuration you might have goes here...
```
Comment (philipphofmann) [inline src/wizard/dotnet/winforms.md:43]: `l`: Maybe an extra new line?
```suggestion
        // Any other configuration you might have goes here...
        
```

File: src/wizard/dotnet/winforms.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+<div class="alert alert-info" role="alert"><h5 class="no_toc">Using .NET Framework prior to 4.6.1?</h5>
+    <div class="alert-body content-flush-bottom">
+        <a href="https://docs.sentry.io/clients/csharp/">Our legacy SDK</a> supports .NET Framework as early as 3.5.
+    </div>
+</div>
+
+## Initialize the SDK
+
+Initialize the SDK as early as possible, like in the constructor of the `App`:
```
Comment (philipphofmann) [inline src/wizard/dotnet/winforms.md:26]: `l`:
```suggestion
Initialize the SDK as soon as possible, like in the constructor of the `App`:
```

File: src/wizard/dotnet/winforms.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+### Documentation
+
+Once you've verified the package is initialized properly and sent a test event, consider visiting our [complete WinForms docs](https://docs.sentry.io/platforms/dotnet/guides/winforms/).
+
+### Samples
+
+You can find an example WinForms app with Sentry integrated [on this GitHub repository.](https://github.com/getsentry/examples/tree/master/dotnet/WindowsFormsCSharp)
+
+See the following examples that demonstrate how to integrate Sentry with various frameworks.
```
Comment (philipphofmann) [inline src/wizard/dotnet/winforms.md:57]: `l`:
```suggestion
See the following examples that demonstrate how to integrate Sentry with various frameworks:
```

File: src/wizard/dotnet/winforms.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        Application.Run(new Form1());
+    }
+}
+```
+
+### Documentation
+
+Once you've verified the package is initialized properly and sent a test event, consider visiting our [complete WinForms docs](https://docs.sentry.io/platforms/dotnet/guides/winforms/).
+
+### Samples
```
Comment (philipphofmann) [inline src/wizard/dotnet/winforms.md:53]: `l`: Maybe it makes sense to also add these two sections to the other wizards for .NET. If you agree I think we should do this in another PR.

File: src/wizard/dotnet/winforms.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+You can find an example WinForms app with Sentry integrated [on this GitHub repository.](https://github.com/getsentry/examples/tree/master/dotnet/WindowsFormsCSharp)
+
+See the following examples that demonstrate how to integrate Sentry with various frameworks.
+
+- [Multiple samples in the `dotnet` SDK repository](https://github.com/getsentry/sentry-dotnet/tree/main/samples) (**C#**)
```
Comment (philipphofmann) [inline src/wizard/dotnet/winforms.md:59]: `l`: Why `(**C#**)`? Is this some special notation or does it render as **C#**?

**Review by philipphofmann — APPROVED** [review]
LGTM, a few minor nitpicks.

File: src/wizard/dotnet/winforms.md (hunk 5 above)
Comment (bruno-garcia) [reply src/wizard/dotnet/winforms.md:60]: **a** 
just makes it bold

File: src/wizard/dotnet/winforms.md (hunk 4 above)
Comment (bruno-garcia) [reply src/wizard/dotnet/winforms.md:54]: Added for WPF and WinForms

**Pull request merged**